staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_test.go

```diff
@@ -1173,21 +1175,23 @@ func Test_growSlice(t *testing.T) {
 		},
 		{
 			name: "with existing capacity above max",
-			args: args{initialCapacity: 12, maxCapacity: 10, sizes: []int{8, 4}},
+			args: args{initialCapacity: 12, initialLen: 5, maxCapacity: 10, sizes: []int{8, 4}},
```

Comment:
Thanks -- can you duplicate these test cases before modifying? That way it's 100% clear that no existing test coverage is lost.

Reply:
Done